src/bacnet/basic/object/netport.c

```diff
@@ -12,6 +12,8 @@
 #include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
+#include <ctype.h>
```

Comment:
config.h is already included in bacdef.h so that it is the first define, and bacdef.h is designated as the first define. Please remove config.h here.

Reply:
done.